src/Composer/Command/DependsCommand.php

```diff
@@ -22,6 +24,14 @@
  */
 class DependsCommand extends BaseDependencyCommand
 {
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor(self::ARGUMENT_PACKAGE)) {
+            $rootPackage = $this->getComposer(true, $input->getOption('no-plugins'))->getPackage();
```

Comment:
This is sufficient/equivalent:

```suggestion
            $rootPackage = $this->getComposer()->getPackage();
```